aabutil.go

```diff
@@ -1,100 +0,0 @@
-package main
```

Comment:
Functions in this file were used from `ensureAABsHasUniversalAPKPair`, which was removed.